### DISCUSSION TITLE
Feature - Related Products

### DISCUSSION THREAD

**Discussion by torvista** [zencart/zencart#6830] [Ideas]
I note that a common feature elsewhere is being able to define products/accessories that could be purchased alongside a primary product.
Normally I manually add these into a product description, and while it is not a feature that's a make or break for me, it seems a significant feature for others.
 
If there is an up to date commercial plugin for this, perhaps it should be referenced on the front/home page. I only found a 2012 one in the Plugins.

**Comment by drbyte** [comment]
Is "cross-sell" another term for what you're referring to? Or distinctly different?

**Reply by torvista** [reply]
I take 
related: an add-on specific to the current product - like a support bracket for a screen
cross-sell: unrelated product that may complement with the current product - shoes to go with a t-shirt
There is also upsell, a better version of the current product.

All of which are common features elsewhere.

I know swguy had a myriad of them...if they are ready for public consumption and they are the only game in town, why not promote them as readily available features on the front page list, maybe even current users are not aware of them?
